content/community/changelog/deployment/migration/_index.nb.md

```diff
@@ -55,7 +55,7 @@ version: 1.1.0
 dependencies:
 - name: deployment
   repository: https://charts.altinn.studio/
-  version: 1.1.0
+  version: 2.5.0
```

Comment:
litt usikker på om denne linjen trengs å justeres. hvis de kjører v 2.5.0 som default så vil de ikke ha det gamle oppsettet eller ha behov for migrering.

Reply:
Jeg tenker det er best at det som står oppdaterte versjonen her. De aller fleste bruker versjon 2.1.0 - det er bare bra om de går inn her at de ser at gjeldende versjon er 2.5.0.